packages/graphql-transformers-e2e-tests/src/__tests__/MutationCondition.e2e.test.ts

```diff
@@ -1371,3 +1385,113 @@ describe(`Deployed Mutation Condition tests`, () => {
     expect(deleteResponse.data.deletePost.version).toEqual(1);
   });
 });
+
+describe(`Local V4-V5 Transformer tests`, () => {
+  it('V4 transform result', () => {
+    const validSchema = `
+            type Post
+            @model
+            {
+                id: ID!
+                content: String
+                rating: Int
+                state: State
+                stateList: [State]
+            }
+
+            enum State {
+              DRAFT,
+              PUBLISHED
+            }
+        `;
+
+    const schema = transformAndParseSchema(validSchema, TRANSFORM_BASE_VERSION);
+
+    const filterType = getInputType(schema, 'ModelPostFilterInput');
+    expectFieldsOnInputType(filterType, ['id', 'content', 'rating', 'state', 'stateList', 'and', 'or', 'not']);
+    doNotExpectFieldsOnInputType(filterType, ['attributeExists']);
+    doNotExpectFieldsOnInputType(filterType, ['attributeType']);
+
+    expectInputTypeUndefined(schema, 'ModelPostConditionInput');
+
+    expectInputTypeDefined(schema, 'ModelStringFilterInput');
+    expectInputTypeDefined(schema, 'ModelIDFilterInput');
+    expectInputTypeDefined(schema, 'ModelIntFilterInput');
+    expectInputTypeDefined(schema, 'ModelFloatFilterInput');
+    expectInputTypeDefined(schema, 'ModelBooleanFilterInput');
+    expectInputTypeDefined(schema, 'ModelStateFilterInput');
+    expectInputTypeDefined(schema, 'ModelStateListFilterInput');
+
+    expectInputTypeUndefined(schema, 'ModelSizeInput');
+    expectEnumTypeUndefined(schema, 'ModelAttributeTypes');
+
+    const mutation = <ObjectTypeDefinitionNode>(
+      schema.definitions.find((def: DefinitionNode) => def.kind === Kind.OBJECT_TYPE_DEFINITION && def.name.value === 'Mutation')
+    );
+    expect(mutation).toBeDefined();
+
+    const checkMutation = (name: string) => {
+      const field = <FieldDefinitionNode>mutation.fields.find(f => f.name.value === `${name}Post`);
+      expect(field).toBeDefined();
+      const conditionArg = field.arguments.find(a => a.name.value === 'condition');
+      expect(conditionArg).toBeUndefined();
+    };
+
+    checkMutation('create');
+    checkMutation('update');
+    checkMutation('delete');
+  });
+
+  it(`V${TRANSFORM_CURRENT_VERSION} transform result`, () => {
```

Comment:
should this be feature version instead of current version

Reply:
Just like for the other test I added a specific e2e for V5 and have a current one as well